cli/src/main/kotlin/com/wire/kalium/cli/ConversationsApplication.kt

```diff
@@ -73,6 +73,9 @@ class ConversationsApplication : CliktCommand() {
     private val cliFileSystem = CliFileSystem(DataStoragePaths(assetsStoragePath, cachePath))
 
     override fun run(): Unit = runBlocking {
+        cliFileSystem.createDirectory(cachePath.value.toPath())
```

Comment:
The `ConversationApplication` is some legacy app which I don't know if we even want to maintain. The CLI app is called `CLIApplication`.

Also would make sense maybe to create the directory somewhere so that we don't have to repeat this setup code in each app which uses Kalium?